django/core/validators.py

```diff
@@ -136,6 +136,12 @@ def __call__(self, value):
                     raise ValidationError(self.message, code=self.code)
             url = value
 
+        if len(urlsplit(value).netloc) > 253:
```

Comment:
No, it is 253, I also took some time to figure that out. The reason is, that DNS internally uses one byte to store the length of the name, see RFC 1035 section 3.1 or https://en.wikipedia.org/wiki/Domain_Name_System#Domain_name_syntax
I'll add a comment, of course.